src/Sulu/Bundle/TagBundle/Controller/TagController.php

```diff
@@ -138,7 +138,8 @@ public function cgetAction(Request $request)
             $fieldDescriptors = $this->fieldDescriptorFactory->getFieldDescriptors('tags');
             $listBuilder = $this->listBuilderFactory->create($this->tagClass);
 
-            $names = array_filter(explode(',', $request->get('names', '')));
+            $names = $this::explodeTags($request->get('names', ''));
```

Comment:
Why not `$this->explodeTags`?

Reply:
This was a static function before ... i was probably a little bit sloppy here